src/common/store-data-to-scan-node-result-converter.ts

```diff
@@ -25,17 +25,21 @@ export type ScanNodeResult = UnifiedResult & {
 
 export type ConvertResultsToCardSelectionStoreDataCallback = (
     state: CardSelectionStoreData,
-    results: ScanNodeResult[],
+    results: ScanNodeResult[] | null,
 ) => CardSelectionStoreData;
 
 export const convertResultsToCardSelectionStoreData: ConvertResultsToCardSelectionStoreDataCallback =
-    (state: CardSelectionStoreData, results: ScanNodeResult[]): CardSelectionStoreData => {
+    (state: CardSelectionStoreData, results: ScanNodeResult[] | null): CardSelectionStoreData => {
         if (results) {
             results.forEach(result => {
                 if (result.status !== 'fail' && result.status !== 'unknown') {
                     return;
                 }
 
+                if (state.rules === null) {
```

Comment:
I'm not sure I follow why this is necessary? This code is invoked in the fast pass (needs review/issues) card selection stores as well and would be hesitant to make any behavioral changes that might impact them.

Reply:
Removed